compressor/tests/test_offline.py

```diff
@@ -62,8 +62,44 @@ def setUp(self):
             'COMPRESS_OFFLINE': True
         }
 
+        if django.VERSION >= (1, 8):
+            override_settings['TEMPLATES'] = [
+                {
+                    "BACKEND": "django.template.backends.django.DjangoTemplates",
+                    "APP_DIRS": True,
+                    "DIRS": override_settings['TEMPLATE_DIRS'],
+                },
+            ]
+
         if "jinja2" in self.engines:
-            override_settings["COMPRESS_JINJA2_GET_ENVIRONMENT"] = lambda: self._get_jinja2_env()
+            if django.VERSION < (1, 8):
+                override_settings["COMPRESS_JINJA2_GET_ENVIRONMENT"] = lambda: self._get_jinja2_env()
+            else:
+                override_settings['TEMPLATES'] += [
```

Comment:
Huh, just watching the WWDC live stream ... though quite boring this year ...

anyway, tried to fix it by sodastsai/django-compressor@986ff0f